README.md

```diff
@@ -272,44 +239,25 @@ package main
 import (
   "io"
   "log"
-  "bufio"
   "fmt"
-  "os"
-  "strings"
 
   "github.com/databricks/databricks-sdk-go"
 )
 
 func init() {
-  // Replace with your own product's name and version.
   databricks.WithProduct("awesome-product", "0.0.1")
-  
-  // Disable printing debug information to output.
-  // Comment the following code to enable printing debug information.
+
   log.SetOutput(io.Discard)
 }
 
 func main() {
   // Perform Azure client secret authentication for a Databricks workspace.
   //  
-  // Choose from one of the following authentication options:
-  //
-  // Option 1: Uncomment the following code and then run. This assumes you have already set
-  // the correct environment variables or .databrickscfg file field equivalents for
+  // This assumes you have already set the correct environment variables or
+  // .databrickscfg file field equivalents for
   // AzureResourceID, AzureTenantID, AzureClientID, and AzureClientSecret.
 
-  // w := databricks.Must(databricks.NewWorkspaceClient(&databricks.Config{AuthType: "azure-client-secret"}))
-
-  // Option 2: To ask the user at run time for the needed information,
-  // uncomment the following code and then run.
-
-  // w := databricks.Must(databricks.NewWorkspaceClient(&databricks.Config{
-  //   AuthType:          "azure-client-secret",
-  //   AzureResourceID:   askFor("Azure resource ID for your Azure Databricks workspace:"),
-  //   AzureTenantID:     askFor("Azure tenant ID for your Azure AD service principal:"),
-  //   AzureClientID:     askFor("Azure client ID for your Azure AD service principal:"),
-  //   AzureClientSecret: askFor("Azure client secret for your Azure AD service principal:"),
-  // }))
+  w := databricks.Must(databricks.NewWorkspaceClient(&databricks.Config{AuthType: "azure-client-secret"}))
```

Comment:
please keep `askFor` lines - we want to document property names :)

Reply:
@nfx Sorry, I'm unclear what you mean by keeping `askFor` lines specifically, here and throughout. Do you prefer I keep the "Option 2" instead of "Option 1?"

Reply:
@PaulCornellDB more like: in specific auth examples (aad spn, gcp oidc, ...) we need to show what configuration properties are at play. In the interactive copy-pastable example. `askFor` is a good way to do that interactively. expect that people would remove that in production code. it's go, programmers are more mature than those using python :D